modules/javafx.graphics/src/main/native-glass/mac/GlassRobot.m

```diff
@@ -443,17 +443,7 @@ - (void)mouseRelease:(UInt32)buttons
         CGImageRef screenImage = CGWindowListCreateImage(bounds, kCGWindowListOptionOnScreenOnly, kCGNullWindowID, kCGWindowImageDefault);
         if (screenImage != NULL)
         {
-            jint pixWidth, pixHeight;
-
-            if (isHiDPI) {
-                pixWidth = (jint)CGImageGetWidth(screenImage);
-                pixHeight = (jint)CGImageGetHeight(screenImage);
-            } else {
-                pixWidth = width;
-                pixHeight = height;
-            }
-
-            jintArray pixelArray = (*env)->NewIntArray(env, (jsize)pixWidth*pixHeight);
+            jintArray pixelArray = (*env)->NewIntArray(env, (jsize)width*height);
```

Comment:
I suspect it is these changes that are causing the scaling bug on Mac with HiDPI. They do not look correct to me.